plugins/CoreConsole/Commands/CoreArchiver.php

```diff
@@ -25,29 +23,20 @@ protected function configure()
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $url = $input->getOption('url') ?: $input->getOption('piwik-domain');
-        if (empty($url)) {
-            throw new \InvalidArgumentException("The --url argument is not set. It should be set to your Piwik URL, for example: --url=http://example.org/piwik/.");
-        }
-
-        if (is_string($url) && $url && in_array($url, array('http://', 'https://'))) {
-            // see https://github.com/piwik/piwik/issues/5180 and http://forum.piwik.org/read.php?2,115274
-            throw new \InvalidArgumentException('No valid URL given. If you have specified a valid URL try --piwik-domain instead of --url');
-        }
-
-        $archiver = self::makeArchiver($url, $input);
-
-        try {
-            $archiver->main();
-        } catch (Exception $e) {
-            $archiver->logFatalError($e->getMessage());
-        }
+        $archiver = self::makeArchiver("", $input);
+        $archiver->main();
     }
 
-    // also used by another console command
+    /**
+     * Used by another console command, so BC must be maintained.
```

Comment:
core:archive is used in cron tabs so that would break a lot of Piwik's I think :).